pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -117,16 +110,16 @@ func (th *TopologyHint) IsEqual(topologyHint TopologyHint) bool {
 // or `a` NUMANodeAffinity attribute is narrower than `b` NUMANodeAffinity attribute.
 func (th *TopologyHint) LessThan(other TopologyHint) bool {
 	if th.Preferred != other.Preferred {
-		return th.Preferred
+		return th.Preferred == true
```

Comment:
Why the explicit check against true?

Reply:
Hmm, yeah strange. It shouldn't be there. Fixed.